docs/source/suite.rst

```diff
@@ -52,6 +85,8 @@ myoHand
 The dexterous human hand requires coordination of multiple highly redundant muscles, which have complementary and antagonistic effects on various joints.
 This musculoskeletal model is comprised of 29 bones, 23 joints, and 39 muscles-tendon units.
 
+Details of this model can be found at `myoHand <https://github.com/MyoHub/myo_sim/tree/dbaecde82a18341661d4eb6d41928f26dc017bdd/hand>`__
```

Comment:
@Vittorio-Caggiano -- 
1. docs are pointing readers to the `myo_sim` repo. Do we want this? 
2. Additionally, they are linking to some weird commit. This should point to latest on `main` 
cc @raku-slyu

Reply:
Updated and point the link to main now

Reply:
resolved following Vittorio's comment